src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -36,6 +36,7 @@
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_PERSON_AMY = "Amy";
```

Comment:
sorry can i check why this was added!!

Reply:
Because in UniqueTagList, there is this test that checks whether the contains(Tag) method returns true even when the tag is edited. So to edit the tag, we will have to edit the person list (by adding one person in) that is attached to this tag. 

@Test
    public void contains_tagWithSameIdentityFieldsInList_returnsTrue() {
        uniqueTagList.add(CS2103T);
        Tag editedCS2103T = new TagBuilder(CS2103T).withPersons(VALID_PERSON_AMY)
                .build();
        assertTrue(uniqueTagList.contains(editedCS2103T));
    }